picasso-loader/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,9 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This XML should be formatted like the app's [manifest](https://github.com/wordpress-mobile/WordPress-Aztec-Android/blob/develop/app/src/main/AndroidManifest.xml).